.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+on: [push, pull_request]
+
+env:
+  TZ: "Europe/Amsterdam"
+  CI: "true"
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        job: [lint, test]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Cache .npm
+        uses: actions/cache@v1
+        env:
+          cache-name: cache-dot-npm
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-build-${{ env.cache-name }}
+
+      - name: Install libgif-dev
+        run: sudo apt install libgif-dev
+
+      - name: Setup Node 11.15
+        uses: actions/setup-node@v1
+        with:
+          node-version: 11.15
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Run ${{ matrix.job }}
+        run: npm run ${{ matrix.job }}
+
+      - name: Upload coverage report
+        uses: actions/upload-artifact@v1
+        if: matrix.job == 'test'
+        with:
+          name: coverage-report
+          path: coverage/
```

Comment:
What does this step do, exactly? Do we need to expose the coverage report?

Reply:
This uploads the coverage report to Github, so you can easily view the report. They are stored for [90 days](https://help.github.com/en/actions/configuring-and-managing-workflows/persisting-workflow-data-using-artifacts#about-workflow-artifacts).

![image](https://user-images.githubusercontent.com/5213690/80454645-a8b09d00-892a-11ea-90d0-42a6120645eb.png)

See: https://github.com/Amsterdam/signals-frontend/runs/622158798

I think exposing the coverage report can be convenient for developers who work on a feature.

Reply:
Indeed. Nice!